t/distmanifest.t

```diff
@@ -7,5 +7,7 @@ BEGIN {
         unless -d '.git' || $ENV{AUTHOR_TESTING};
 }
 
-use Test::DistManifest;
+eval { require Test::DistManifest; };    ## no critic (ErrorHandling::RequireCheckingReturnValueOfEval)
+plan skip_all => 'Test::DistManifest required' if $@;
```

Comment:
This is not what we want.  When a developer is releasing the code, we *need* this test to be run, to ensure the MANIFEST is up to date. Allowing the test to be skipped defeats the whole point.